crates/loans/src/lib.rs

```diff
@@ -437,10 +437,6 @@ pub mod pallet {
 
     #[pallet::hooks]
     impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
-        fn on_runtime_upgrade() -> frame_support::weights::Weight {
```

Comment:
I'd have removed the whole hooks implementation, rn it still has pre and post hooks. But whatever, I don't think it needs to block merging